src/openrct2/ride/TrackDesign.cpp

```diff
@@ -2111,7 +2112,9 @@ void track_design_draw_preview(TrackDesign* td6, uint8_t* pixels)
  */
 static map_backup* track_design_preview_backup_map()
 {
-    map_backup* backup = static_cast<map_backup*>(malloc(sizeof(map_backup)));
+    // map_backup* backup = static_cast<map_backup*>(malloc(sizeof(map_backup)));
+    auto backup_ptr = std::make_unique<map_backup>();
```

Comment:
You don't need to create a new variable name `backup` can still be accessed just the same.